ui/components/requirements.jsx

```diff
@@ -47,7 +49,12 @@ function Requirements({ keywords, pagedReqs, policies, router }) {
         </div>
         <ul className="list-reset">
           { pagedReqs.results.map(requirement =>
-            <Requirement key={requirement.req_id} requirement={requirement} />) }
+            <Requirement
+              key={requirement.req_id}
+              requirement={requirement}
+              sunset={requirement.policy.sunset}
```

Comment:
It looks like we're accessing the sunset date via `requirement` in the `Requirement` component; does that mean we don't need to pass it in separately here?